citar.el

```diff
@@ -903,17 +918,21 @@ into the corresponding reference key.  Return
            (lambda (key-entry)
              (citar-get-link (cdr key-entry)))
            key-entry-alist))
-         (resource-candidates (delete-dups (append files (remq nil links))))
-         (resources
-          (when resource-candidates
-            (completing-read-multiple "Related resources: " resource-candidates))))
-    (if resource-candidates
-        (dolist (resource resources)
-          (cond ((string-match "http" resource 0)
-                 (browse-url resource))
-                (t (citar-file-open resource))))
+         (selection (citar-select-resource files links)))
+    (if files
+        (cond ((string-match "http" selection 0)
+               (browse-url selection))
+              (t (citar-file-open selection)))
       (message "No associated resources"))))
 
+(defun citar-open-multi (selection)
```

Comment:
This command is unnecessary and should be removed. Embark already dispatches consult-multi to file and url. That's the purpose of consult-multi.